web/models.py

```
@@ -1752,6 +1752,65 @@ def display_name(self):
+
+class MeetupRegistration(models.Model):
+    meetup = models.ForeignKey(Meetup, on_delete=models.CASCADE)
+    user = models.ForeignKey(User, on_delete=models.CASCADE)
+    registered_at = models.DateTimeField(auto_now_add=True)
+
+    def __str__(self):
+        return f"{self.user.username} registered for {self.meetup.title}"
+
+    class Meta:
+        unique_together = ["user", "meetup"]
+        ordering = ["-registered_at"]
```

Ruff (0.8.2)

Order of model's inner classes, methods, and fields does not follow the Django Style Guide: `Meta` class should come before `__str__` method

(DJ012)